sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_async.py

```diff
@@ -182,7 +183,8 @@ def _next_message_in_buffer(self):
         # pylint:disable=protected-access
```

Comment:
Could we please remove the comment on line 180? As this is no longer only used by the uamqp transport :)